swig/python/flamegpu.i

```diff
@@ -12,6 +12,7 @@
 #pragma SWIG nowarn=362
 // operator++ ignored (not supported by python)
 #pragma SWIG nowarn=383
+#pragma SWIG nowarn=384
```

Comment:
Comment would be good. is this `operator--`?

Reply:
That rings a bell, but I can't confirm at this point.